lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -10,7 +10,7 @@
 import boto3
 
 logging.getLogger().setLevel(logging.DEBUG)
-logger = logging.getLogger()
+logger = logging.getLogger("va-notify-vetext-forwarding-lambda")
 logger.setLevel(logging.DEBUG)
 
 def vetext_incoming_forwarder_lambda_handler(event: any, context: any):
```

Comment:
I'll change this to dict here because we're expecting a dictionary.  But, just as an FYI, the AWS docs say that event can be a variety of things depending on what triggered it: 

"It is usually of the Python dict type. It can also be list, str, int, float, or the NoneType type." - AWS docs

Reply:
My understanding is that it is always an event dictionary when the lambda trigger is an API gateway.
https://docs.aws.amazon.com/lambda/latest/dg/services-apigateway.html#apigateway-example-event